src/course-unit/hooks.jsx

```diff
@@ -202,3 +222,18 @@ export const useCourseUnit = ({ courseId, blockId }) => {
     handleNavigateToTargetUnit,
   };
 };
+
+export const useLayoutGrid = (unitCategory, isUnitLibraryType) => (
```

Comment:
Please add a `/** JSDoc string */` explaining what this is and why we have it. It's not very clear.

Reply:
Added